modules/user_manual/pages/files/manual_file_locking.adoc

```diff
@@ -9,7 +9,7 @@ All storages are supported as locking takes place on the WebDAV level. The locks
 
 == Using Manual File Locking
 
-NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud admin, because this feature allows users to lock other users files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock until the lock expires.
+NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud admininstrator, because this feature allows users to lock other users files *exclusively*. Lock releasing can only be done by the locking user, a user being member of a lock breaking group if defined by the ownCloud administrator or until the lock expires.
```

Comment:
```suggestion
NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud administrator, because this feature allows users to lock other users files *exclusively*. Lock releasing can only be done by the locking user, a user who is a member of a lock breaking group (if defined by the ownCloud administrator) or until the lock expires.
```